storage/src/main/java/tech/pegasys/teku/storage/server/Database.java

```diff
@@ -101,6 +103,12 @@ public interface Database extends AutoCloseable {
   @MustBeClosed
   Stream<SignedBeaconBlock> streamFinalizedBlocks(UInt64 startSlot, UInt64 endSlot);
 
+  @MustBeClosed
+  Stream<SignedBeaconBlock> streamHotBlocks();
+
+  @MustBeClosed
+  Stream<ColumnEntry<Bytes32, CheckpointEpochs>> streamCheckpointEpochs();
```

Comment:
We probably shouldn't be exposing the kvstore `ColumnEntry` as part of the generic `Database` implementation.  Could just use `Map.Entry` interface though since `ColumnEntry` already implements it and it's generic enough.